app/views/questions/index.html.haml

```diff
@@ -1,6 +1,7 @@
 %h1 Listing questions
 
-%table#question-table
+%table#questiontable
```

Comment:
You should preserve this change; we've been separating classes and ids with dashes. Also, if you `grep` for it, some Javascript uses this id.